pkg/apis/virt/v1alpha1/types.go

```diff
@@ -67,9 +67,24 @@ type CPU struct {
 }
 
 type Memory struct {
-	Size *resource.Quantity `json:"size,omitempty"`
+	Size      *resource.Quantity `json:"size,omitempty"`
+	Hugepages *Hugepages         `json:"hugepages,omitempty"`
 }
 
+type Hugepages struct {
+	// +kubebuilder:default="1Gi"
+	PageSize HugepagesSize `json:"pageSize,omitempty"`
+}
+
+// +kubebuilder:validation:Enum="2Mi";"1Gi"
```

Comment:
we can omit the double quotes here?

Reply:
controller-gen will report errors without quotes here, maybe it can not determine  the type of object that has a  mixture of numbers and string.